modules/nw-ingress-edge-route-default-certificate.adoc

```diff
@@ -0,0 +1,51 @@
+// This is included in the following assemblies:
+//
+// networking/routes/route-configuration.adoc
+
+:_content-type: PROCEDURE
+[id="creating-edge-route-with-default-certificate_{context}"]
+= Creating a route using the default certificate through an ingress object
```

Comment:
Should Ingress be capitalized here? I see in another heading in this assembly, in the same context as this, it _is_ capitalized...but the casing changes all the time throughout the docs generally 😕 @kalexand-rh do you have an opinion on this particular case (no pun intended)?

Reply:
[Ingress](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#ingress) is capitalized.